src/core/MIGRATION_EXAMPLES.md

```diff
@@ -1178,6 +1180,28 @@ const request = client.asCurrentUser.ping({}, {
 });
 ```
 
+- the new client doesn't provide exhaustive typings for the response object yet. You might have to copy
+response type definitions from the Legacy Elasticsearch library until https://github.com/elastic/elasticsearch-js/pull/970 merged.
+
+```ts
+// platform provides a few typings for internal purposes
+import { SearchResponse } from 'src/core/server';
+const { body } = await client.search<SearchResponse<any>>(...)
```

Comment:
Could we show an example of using `SearchResponse`'s generic to specify the body shape?

Reply:
Added. Let me know if you have another example in your mind.